src/main/java/Exam.java

```diff
@@ -0,0 +1,13 @@
+
+import java.util.Scanner;
```

Comment:
why this import here.

Reply:
removed the import statement ,that is due to previous code copied